okhttp/src/main/kotlin/okhttp3/internal/platform/Platform.kt

```diff
@@ -204,56 +206,66 @@ open class Platform {
 
     /** Attempt to match the host runtime to a capable Platform implementation. */
     private fun findPlatform(): Platform {
-      val android10 = Android10Platform.buildIfSupported()?.also {
-        AndroidLog.enable()
-      }
+      // This explicit check avoids activating in Android Studio with Android specific classes
+      // available when running plugins inside the IDE.
+      if (isAndroid) {
```

Comment:
I think the body of this can be simplified to
```
if (isAndroid) {
  AndroidLog.enable()
  return Android10Platform.buildIfSupported() ?: AndroidPlatform.buildIfSupported()!!
}
```
Do we ever expect the base Android platform to fail if we detected that we're on it?